### ISSUE TITLE
FPS display position should be set by API

### ISSUE THREAD

**Issue by aismann** [adxeproject/adxe#695]
FPS display position should be set by API
Reason: The default FPS display position (LEFTBOTTOM) is not always the best to read the FPS values.

Example:
```
            Director::getInstance()->setFPSPos(Director::FPSPosition::RIGHTMIDDLE);
```

```
    enum FPSPosition
    {
        TOP_LEFT       = 7,
        TOP_CENTER     = 8,
        TOP_RIGHT      = 9,
        CENTER_LEFT    = 4,
        CENTER         = 5,
        CENTER_RIGHT   = 6,
        BOTTOM_LEFT    = 1,
        BOTTOM_CENTER  = 2,
        BOTTOM_RIGHT   = 3
    } _FPSPosition;
```
Hint: the position/numbers be like the position on the numpad for easier remind ;)

Test on test_cpp:  19:EventDispatcher: 5:Label Receives Keyboard Event
![image](https://user-images.githubusercontent.com/8652787/176916292-56912429-85dd-4f2c-9497-b0bad6fdca54.png)

**Comment by aismann** [comment]
PR is ready to push.

**Comment by aismann** [comment]
Implemented with #696
